### PR TITLE
Budget App Rails

### PR DESCRIPTION
**Pull request by kessio**
Hello 👋🏾 
The following has been implemented on this Pull request:
Create Ruby on Rails application that allows the user to:

- [x] register and log in, so that the data is private to them.
- [x] introduces new transactions associated with a category.
- [x] see the money spent on each category.
- [x] Linters have passed

### CODE REVIEW

File: app/views/expenses/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  <div>
+    <%= form.text_field :name, placeholder: "Name", class:"form-control py-3 mt-3" %>
+  </div>
+
+  <div>
+    <%= form.text_field :amount, placeholder: "Amount", class:"form-control py-3 mt-3" %>
+  </div>
+
+  <div>
```
Comment (fobadara) [inline app/views/expenses/new.html.erb:19]: [REQUIRED] Kindly include an input field to select which category a new expense belongs to

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @kessio, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing
✅ Good first attempt

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/expenses/new.html.erb (hunk 1 above)
Comment (kessio) [reply app/views/expenses/new.html.erb:19]: Hello @fobadara , thanks for the review. The new expense is added under a category, so when the form is submitted the category id on the URL is saved in the database. I did not see a need to have a category field yet the transaction is already under a category. Thanks

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @kessio 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.
- Your linters are passing all the checks.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**